.github/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ## 0.10.0
 
+1. [MCDU] Reset of flight plan and stored data on Database switch - @Maverickwoe (Garoomf#4683)
```

Comment:
Please put at the bottom of the 0.10.0 list.

Reply:
Done